site/content/contribute/getting-started/contribution-checklist.md

```diff
@@ -23,8 +23,10 @@ Follow this checklist for submitting a pull request (PR):
     - [../mattermost-mobile../en.json](https://github.com/mattermost/mattermost-mobile/blob/master/assets/base/i18n/en.json)
 6. PR is submitted against `master` branch.
 7. PR title begins with the JIRA or GitHub Ticket ID (e.g. `MM-394` or `GH-394`) and summary template is filled out.
-8. Once submitted, the automated build process must pass for the PR.
+8. If your PR adds or changes a RESTful API endpoint, please update the [API documentation](https://github.com/mattermost/mattermost-api-reference).
```

Comment:
My apologies, I'm reading the list from start to finish and noticed the sentence structure for 8 + 9 is different..so propose

```
8. If your PR adds or changes a RESTful API endpoint, [API documentation](https://github.com/mattermost/mattermost-api-reference) is updated.
9. If you PR adds a new plugin API method or hook, an example is included in the [Plugin Starter Template](https://github.com/mattermost/mattermost-plugin-sample).
```

Also, wondering if step 10 should say `Once submitted, the automated build process must pass for all PRs.`

Reply:
While I see the point of keeping this consistent, there is a difference between 8. and 9. and the other items. All other items should be checked before or while submitting the PR. But 8. and 9. should be taken care of after the PR has been merged or at least after the first round of review was positive. They should be more of a heads up, that the PR to the other repos should be done at some point.

Reply:
That's a good point. Should we then move step 10 ahead of these new steps?

Because right now, as written, it looks like 8 + 9 are required as part of the PR submission process.

Reply:
:+1: on moving them before 10. Would you take a second look please?